optimum/exporters/openvino/__main__.py

```diff
@@ -336,6 +344,15 @@ class StoreAttr(object):
             generation_config.save_pretrained(output)
         maybe_save_preprocessors(model_name_or_path, output)
 
+        if tokenizer is not None and is_openvino_tokenizers_available():
+            try:
+                export_tokenizer(tokenizer, output)
+            except Exception as exception:
```

Comment:
Would prefer to add an argument to trigger this export

Reply:
Add `--convert-tokenizer` Option to CLI

Reply:
thanks a lot @apaniukov, after our internal discussion I'm OK with both solutions so we can either revert https://github.com/huggingface/optimum-intel/pull/513/commits/09b067fa1245cc12ce7117174fb9c8a114118380 or keep `convert-tokenizer` (+ some tests needs to be fixed), let me know what work best for you and will merge afterwards

Reply:
Thanks, resolved merge conflicts and pushed a fix for the tests.

Reply:
@apaniukov, consider creating a PR with removing the mandatory `--convert-tokenizer`. Instead of enabling conversion we probably need a key to disable conversion, something like `--disable-convert-tokenizer`. There are options for how to deprecate the old option:

1. Remove the old key without any trace. Old users who hard-code somewhere this key will see the error about an unknown option.
2. Keep the old key, print a warning that explains that this key is not needed anymore because tokenizers are converted by default, and instead of enabling conversion now you can disable it if something really bad happens in that part. So explain the change with this key in all details. But conversion goes as it should go, tokenizers are converted, the model is converted, and that message is just a warning.
3. The same as [2] but the message is an error instead of a warning, the export fails, and nothing is produced as a result.

I like option [2] considering I don't know the level of adoption of the existing key. But if there is evidence that nobody has started using `--convert-tokenizer`, option 3 is better, and if we are bold enough, go with option [1].

Reply:
https://github.com/huggingface/optimum-intel/pull/580